packages/graphql-tool-utilities/config.ts

```diff
@@ -30,6 +32,20 @@ export function getGraphQLSchemaPaths(config: GraphQLConfig) {
   }, []);
 }
 
+export async function getGraphQLProjectIncludedFilePaths(
+  projectConfig: GraphQLProjectConfig,
+) {
+  return (await Promise.all(
+    projectConfig.includes.map((include) =>
+      promisify(glob)(getGraphQLFilePath(projectConfig, include), {
+        ignore: projectConfig.excludes.map((exclude) =>
+          getGraphQLFilePath(projectConfig, exclude),
+        ),
+      }),
+    ),
+  )).flatMap((filePaths) => filePaths);
```

Comment:
This is available in our version of Node?!

Reply:
good question. it's actually getting polyfilled by `apollo-codegen-core`. Should we maybe poly that in more properly?

Reply:
correction. `core-js/es7/array` is pulling in the implementation.

Reply:
I think I should have added `esnext.array` to `tsconfig.json` libs a while back. `apollo-codegen-core` just happens to also include a `flatMap` type def.